recipes/labjackpython/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "LabJackPython" %}
+{% set org = "labjack" %}
+{% set upstreamversion = "4-24-2014" %}
+{% set version = "20140424" %}
```

Comment:
I'm probably too lazy to be that clever. Would just do something like `"."join(upstreamversion.split("-")[::-1])` in Python. Probably needs some tweaks for Jinja. Having `.` are very important as it makes this [PEP 440]( https://www.python.org/dev/peps/pep-0440/#support-for-date-based-version-identifiers ) compatible. Also it may make pinning and version comparison easier later as `conda` supports this..